src/core/mesh/qgsmeshtracerenderer.cpp

```diff
@@ -604,6 +653,11 @@ bool QgsMeshStreamField::isTraceOutside( const QPoint &pixel ) const
   return true;
 }
 
+void QgsMeshStreamField::setMinimizeFieldSize( bool minimizeFieldSize )
+{
+  mMinimizeFieldSize = minimizeFieldSize;
```

Comment:
shouldn't it be named minimumFieldSize..?

Reply:
It's to tell the class to minimize the field size if needed, e.g. to put it's extent on the mesh extent if this one does not fill the device extent. So when the class is used for render on screen, it speed up the rendering, and when the class is used for frame image, the outputsize is on the size of the device.

Reply:
my. comment is only about the choice of wording.. minimum vs mimize